code_generation/systematics.py

```diff
@@ -30,35 +30,38 @@
 
 
 class SystematicShift(object):
-    """
-    Class containing systematic shifts. A systematic shift is a variation of a
-    set of configuration parameters. A dummy shift looks like this:
-    SystematicShift(
-            name="shiftname",
-            shift_config={
-                ("scope1", "scope2"): {
-                    "paramter1": True,
-                    "paramter2": 42.0,
+    """Class containing systematic shifts. A systematic shift is a variation of a set of configuration parameters.
+
+    A dummy shift looks like this::
```

Comment:
typo "::"

Reply:
This is doctring formatting, so the block below is rendered as a codeblock in the documentation